src/Model/ModelManager.php

```diff
@@ -349,7 +349,7 @@ public function batchDelete(string $class, BaseProxyQueryInterface $query): void
             $entityManager = $this->getEntityManager($class);
 
             $i = 0;
-            foreach ($query->execute() as $object) {
+            foreach ($query->getDoctrineQuery()->toIterable() as $object) {
```

Comment:
this previously was `$query->getQueryBuilder()->getQuery()->toIterable()`. The `getDoctrineQuery()` docs says its better to use it in favour of the `getQueryBuilder()->getQuery()``, is it correct @VincentLanglet ?

Reply:
The DoctrineQuery will add the "correct" sortBy and sortOrder.
It's useless here, but won't hurt. So we can be consistent.